rpm/generic/zfs.spec.in

```diff
@@ -33,20 +33,16 @@
 %global _initconfdir /etc/sysconfig
 %endif
 
+# Set the default python pythonsitedir when undefined.a
+%if %{undefined _pythonsitedir}
+%global _pythonsitedir %{python_sitelib}
```

Comment:
If you set `%__python` to the correct interpreter, then `%python_sitelib` will give you the correct value for a given interpreter environment. That's probably simpler than this odd setup...

Reply:
Good idea